src/Microsoft.DotNet.Build.Tasks/GetNetCoreAppVersionsFromFile.cs

```diff
@@ -0,0 +1,58 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using System.IO;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.DotNet.Build.Tasks
+{
+    public class GetNetCoreAppVersionsFromFile : Task
+    {
+        [Required]
+        public string PathToVersionsFile { get; set; }
+
+        [Output]
+        public string CorefxHash { get; set; }
```

Comment:
Apart from the standard hash, do we expect anything else going into that versions.txt file? If not, I wonder if it would be better to just add one more hardcoded one for standard. I'm happy to do it generic as well, I just want to understand a bit more what do we expect to use this task for.